osu.Game.Tests/Visual/SongSelectV2/TestSceneBeatmapMetadataWedge.cs

```diff
@@ -148,6 +149,38 @@ public void TestTruncation()
             });
         }
 
+        [Test]
+        public void TestOnlineAvailability()
+        {
+            AddStep("online beatmapset", () =>
+            {
+                var (working, onlineSet) = createTestBeatmap();
+
+                currentOnlineSet = onlineSet;
+                Beatmap.Value = working;
+            });
+            AddAssert("success rate populated", () => this.ChildrenOfType<BeatmapMetadataWedge.SuccessRateDisplay>().Single().Data == (4567, 10000));
+            AddAssert("user rating populated", () => this.ChildrenOfType<BeatmapMetadataWedge.UserRatingDisplay>().Single().Data.Length > 0);
+            AddStep("online beatmapset with local diff", () =>
+            {
+                var (working, onlineSet) = createTestBeatmap();
+
+                onlineSet.Beatmaps = Array.Empty<APIBeatmap>();
```

Comment:
What is even going on here??

Reply:
Quick method to populate a beatmapset with no difficulty matching the active working beatmap. I hope an inline comment will be enough for this.

Reply:
No, it's not. I don't get it. If you want to show a "local" beatmap wouldn't it be able changing the `OnlineID` on the beatmap, rather than removing beatmaps from a beatmap set??

Reply:
Sure.

Reply:
https://github.com/ppy/osu/pull/32957/commits/b7830361787d9bdc6fe1de60ac5946be56a88c19

Reply:
Is there a reason you're using `+ 1` rather than `-1`? Or just calling `ResetOnlineInfo`?